blog_system/settings.py

```diff
@@ -0,0 +1,138 @@
+"""
+Django settings for blog_system project.
+
+Generated by "django-admin startproject" using Django 4.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / "subdir".
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-nt0v6+mp7(09-35!)tx88xii(ipi$p2ie(2pjdzorw9basa=h("
+
+# SECURITY WARNING: don"t run with debug turned on in production!
+DEBUG = True
```

Comment:
DEBUG should not be set to True in a production environment as it can expose sensitive information. Consider using environment variables to manage the DEBUG setting.